src/random.cpp

```diff
@@ -451,7 +451,7 @@ static void SeedFast(CSHA512& hasher) noexcept
 
 static void SeedSlow(CSHA512& hasher) noexcept
 {
-    unsigned char buffer[32];
+    unsigned char buffer[32] = {0};
```

Comment:
@practicalswift once I fixed this one, it ran into unrelated   `use-of-uninitialized-value` problems. That could mean this is the only problem, or the other ones just didn't get a chance to get called before the sanitizer bailed out.